app/user/tests/test_views.py

```diff
@@ -99,3 +105,42 @@ def test_invalid_token(self) -> None:
 
         self.assertEqual(resp.status_code, 400)
         self.assertIn("Invalid or expired token", str(resp.data))  # type: ignore[attr-defined]
+
+
+class ProfileTest(APITestCase):
+    def setUp(self) -> None:
+        self.user_test = User.objects.create_user(**test_user)
+
+        self.client = APIClient()
+
+    @patch(
+        "cloudinary.uploader.upload_resource", return_value=fake.image_url()
+    )
+    def test_profile_update(self, upload_resource: None) -> None:
+        Profile.objects.create(user=self.user_test)
+        login_url = reverse("login")
```

Comment:
check on having this in a separate function that you can call whenever you need the token

Reply:
Yes it is necessary,